misc/output_test.py

```diff
@@ -449,6 +449,40 @@ def test_tool_compdb_targets(self) -> None:
             self.assertEqual(expected, actual)
 
 
+    def test_tool_multi_inputs(self) -> None:
+        plan = '''
+rule cat
+  command = cat $in $out
+build out1 : cat in1
+build out2 : cat in1 in2
+build out3 : cat in1 in2 in3
+'''
+        self.assertEqual(run(plan, flags='-t multi-inputs out1'),
+'''out1	in1
```

Comment:
Can you avoid using embedded tabs for readability? For example, replace them in with "<TAB>" in the result before the comparison, as in:

```
self.assertEqual(run(...).replace("\t", "<TAB>"), ''out1<TAB>in1
''')
```

Reply:
changed to **replace("<TAB>", "\t")** instead ;-)